src/main/java/com/minecolonies/coremod/tileentities/ScarecrowTileEntity.java

```diff
@@ -119,87 +117,16 @@
     /**
      * Inventory of the field.
      */
-    private final IItemHandlerModifiable inventory = new IItemHandlerModifiable() {
```

Comment:
As far as I know we should not be using IInventory anymore and instead use IItemhandlers.

Reply:
Ah, true true. There was still one of the functions causing a glitch in the implementation that was there before. I believe it is fixed now. Not quite sure which part was causing it still.